examples/todos/package.json

```diff
@@ -5,29 +5,27 @@
   "keywords": [],
   "main": "src/index.tsx",
   "dependencies": {
-    "@babel/runtime": "7.11.2",
     "@react-spring/web": "9.0.0-rc.3",
-    "antd": "4.6.4",
-    "jotai": "0.5.2",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-scripts": "3.4.3"
+    "antd": "4.9.1",
+    "jotai": "0.11.2",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-scripts": "4.0.1"
   },
   "devDependencies": {
-    "customize-cra": "^1.0.0",
-    "customize-cra-react-refresh": "^1.1.0",
-    "raw-loader": "^4.0.1",
-    "react-app-rewired": "^2.1.6",
-    "@types/react": "16.9.19",
-    "@types/react-dom": "16.9.5",
-    "typescript": "3.7.5",
-    "serve": "^11.3.2"
+    "customize-cra": "1.0.0",
+    "react-app-rewired": "2.1.6",
+    "serve": "11.3.2"
+  },
+  "resolutions": {
+    "webpack": "~4.43.0"
```

Comment:
Webpack resolution workaround for https://github.com/pmndrs/react-spring/issues/1078